policies/roxtl/vulnerabilities-check.rego

```diff
@@ -0,0 +1,130 @@
+package required_checks
+
+import future.keywords.if
+
+# Severity mapping for roxctl
+# CRITICAL_VULNERABILITY_SEVERITY -> Critical
+# IMPORTANT_VULNERABILITY_SEVERITY -> High  
+# MODERATE_VULNERABILITY_SEVERITY -> Medium
+# LOW_VULNERABILITY_SEVERITY -> Low
+
+# Function to get components with vulnerabilities of specific severity that have fixes
+roxctl_get_patched_vulnerabilities(input_data, severity) := vulnerabilities if {
+  vulnerabilities := [{"name": component.name, "version": component.version, "location": location, "vulnerabilities": vuln} |
+    component := input_data.scan.components[_]
+    location := object.get(component, "location", "")
+    vuln := {v.cve | v := component.vulns[_]; v.severity == severity; v.fixedBy != ""}
```

Comment:
Not sure if this condition is aplicable on roxctl outputs, can you confirm/deny this field is present in roxctl scan output?

Reply:
Thats a great news! Thanks!